src/styles/blocks/card.scss

```diff
@@ -0,0 +1,87 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  border: 1px solid $white-border-color;
+  border-radius: $card-border-radius;
+  width: $card-width;
+
+  &__info-logo {
+    width: 160px;
+    height: 134px;
+    padding: 0 3px;
+    margin-bottom: 40px;
+  }
+
+  &__info-title {
+    color: $dark-color;
+    font-weight: 500;
+    font-size: 12px;
+
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__info-code {
+    font-size: 10px;
+    line-height: 14px;
+    color: $grey-color;
+    margin-bottom: 16px;
+  }
+
+  &__review {
+    @extend %flex-between-center;
+
+    line-height: 14px;
+    margin-bottom: 24px;
+
+    &-mark {
+      font-size: 10px;
+      line-height: 14px;
+    }
+  }
```

Comment:
`-mark` is not an element or modifier, so it's a wrong usage of nesting. Although it provides ability to store some class name as variable, we shouldn't use it for class name combining (it's not clear if it's nested class, or just other class with name that includes this word)